int64validator/at_least_sum_of_test.go

```diff
@@ -0,0 +1,100 @@
+package int64validator
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+)
+
+func TestAtLeastSumOfValidator(t *testing.T) {
+	t.Parallel()
+
+	type testCase struct {
+		val                  attr.Value
+		attributesToSumPaths []*tftypes.AttributePath
+		requestConfigRaw     map[string]tftypes.Value
+		expectError          bool
+	}
+	tests := map[string]testCase{
+		"not an Int64": {
+			val:         types.Bool{Value: true},
+			expectError: true,
+		},
+		"unknown Int64": {
+			val: types.Int64{Unknown: true},
+		},
+		"null Int64": {
+			val: types.Int64{Null: true},
+		},
+		"valid integer as Int64 less than sum of attributes": {
+			val: types.Int64{Value: 10},
+			attributesToSumPaths: []*tftypes.AttributePath{
+				tftypes.NewAttributePath().WithAttributeName("one"),
+				tftypes.NewAttributePath().WithAttributeName("two"),
+			},
+			requestConfigRaw: map[string]tftypes.Value{
+				"one": tftypes.NewValue(tftypes.Number, 15),
+				"two": tftypes.NewValue(tftypes.Number, 15),
+			},
```

Comment:
Drive-by note: It'd be great to ensure there are unit tests covering when the one/two attributes are null and/or unknown as well. Or all three are null/unknown.

Reply:
Have added some additional tests.

Reply:
Have added a test case for attributePath(s) referencing unexpected types in `at_least|at_most|equal_to_sum_of_test`.